### PR TITLE
Added taco multiplier on Tuesdays for issue #668

### PR DESCRIPTION
**Pull request by landoniwnl**
I want more tacos

### CODE REVIEW

File: src/services/point.service.ts [hunk 1]
```diff
@@ -15,6 +15,10 @@ export class PointService {
       return;
     }
 
+    if (new Date().getDay() === 2) {
+      amount = amount * 2;
+    }
```
Comment (Armintron) [inline src/services/point.service.ts:20]:
```suggestion
   // give bonus points on tuesdays
    if (new Date().getDay() === 2) {
      amount *= 2;
    }
```

**Review by Armintron — CHANGES_REQUESTED** [review]
Add comments to your code and use a big boi operator for self multiplcation

File: src/services/point.service.ts (hunk 1 above)
Comment (c650) [reply src/services/point.service.ts:20]: Is day zero based or one based? Does it start with Sunday or Monday?

Comment (Armintron) [reply src/services/point.service.ts:20]: > 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getDay
It starts with Sunday = 0

**Review by tanndlin — CHANGES_REQUESTED** [review]
This should be separate from how gambling awards points in some way.
As is, it would make gambling print money

**Comment by tanndlin** [comment]
> This should be separate from how gambling awards points in some way. As is, it would make gambling print money

Like double the amount in the `point_message.handler.ts` and/or `point_react.handler.ts`

**Comment by Armintron** [comment]
> This should be separate from how gambling awards points in some way. As is, it would make gambling print money

That is inherient to any sort of multiplier. Maybe 2x is too much? Just 1.5x? At the end of the day its just the fun of it - its alright if you can more easily make tacos.

**Comment by tanndlin** [comment]
> > This should be separate from how gambling awards points in some way. As is, it would make gambling print money
> 
> That is inherient to any sort of multiplier. Maybe 2x is too much? Just 1.5x? At the end of the day its just the fun of it - its alright if you can more easily make tacos.

Its not really gambling if ur guaranteed to make money

**Comment by Armintron** [comment]
> > > This should be separate from how gambling awards points in some way. As is, it would make gambling print money
> > 
> > 
> > That is inherient to any sort of multiplier. Maybe 2x is too much? Just 1.5x? At the end of the day its just the fun of it - its alright if you can more easily make tacos.
> 
> Its not really gambling if ur guaranteed to make money

In the long run and in the limit to infinity yes, but you can still lose a lot with a big bet. Yes, you could make that back if you sit and spam but at the end of the day, you are driving attraction towards to plugin - which is almost always the end goal.

**Comment by tanndlin** [comment]
> In the long run and in the limit to infinity yes, but you can still lose a lot with a big bet. Yes, you could make that back if you sit and spam but at the end of the day, you are driving attraction towards to plugin - which is almost always the end goal.
 
 Being able to min bet and get back 4x the min bet just ruins the leaderboard, thats inflation man

**Comment by Armintron** [comment]
> > In the long run and in the limit to infinity yes, but you can still lose a lot with a big bet. Yes, you could make that back if you sit and spam but at the end of the day, you are driving attraction towards to plugin - which is almost always the end goal.
> 
> Being able to min bet and get back 4x the min bet just ruins the leaderboard, thats inflation man

So then how does your proposed change fix this? I didnt understand that comment

**Comment by tanndlin** [comment]
> So then how does your proposed change fix this? I didnt understand that comment

Just dont affect the gambling amount

**Comment by Armintron** [comment]
> > So then how does your proposed change fix this? I didnt understand that comment
> 
> Just dont affect the gambling amount

But thats the whole point of this PR. Was your note just on the correct way to alter an award amount? Im not following

**Comment by tanndlin** [comment]
Just change [this](https://github.com/cs-discord-at-ucf/lion/blob/c75a9a0239e7ace53ad2d6863db23ee911a6b9e6/src/app/handlers/point_message.handler.ts#L17) lol

The gamble uses the point service `awardPoints`, so just don't change that

**Comment by Armintron** [comment]
> Just change [this](https://github.com/cs-discord-at-ucf/lion/blob/c75a9a0239e7ace53ad2d6863db23ee911a6b9e6/src/app/handlers/point_message.handler.ts#L17) lol
> 
> The gamble uses the point service `awardPoints`, so just don't change that

I had no clue you were trying to say "we should double the amount users get for sending messages, not for gambling". I get it now. Yes that is a good middle ground

**Comment by tanndlin** [comment]
> Like double the amount in the point_message.handler.ts and/or point_react.handler.ts

**Comment by Armintron** [comment]
> > Like double the amount in the point_message.handler.ts and/or point_react.handler.ts

I understand but as a non-contributer idk what those things do until i look at them

File: src/services/point.service.ts (hunk 1 above)
Comment (Armintron) [reply src/services/point.service.ts:20]: we not doubling the amount for gambling

**Comment by landoniwnl** [comment]
I didn't realize that what I did affected the gambling amount too. It should only give bonus points from messages now.

**Review by tanndlin — APPROVED** [review]
lgtm
we can change this number later if we feel it doesn't have enough impact

**Pull request merged**